packages/eip-5792-middleware/src/hooks/getCapabilities.ts

```diff
@@ -106,15 +109,22 @@ export async function getCapabilities(
     }
 
     const status = isSupported ? 'supported' : 'ready';
+    const hexChainId = chainId as Hex;
 
-    if (acc[chainId as Hex] === undefined) {
-      acc[chainId as Hex] = {};
+    if (acc[hexChainId] === undefined) {
+      acc[hexChainId] = {};
     }
```

Comment:
no need to fix this, but i feel like we can just assume or coerce `chainIdsNormalized` to be a unique list and not have to do this undefined check here